contracts/PangolinExchange.sol

```diff
@@ -29,8 +42,8 @@ contract PangolinExchange is Ownable, IAssetsExchange, ReentrancyGuardUpgradeabl
 
   function refundAvaxBalance() private {
```

Comment:
Why cannot we simplify that and use simply transfer inside the buy method?

Reply:
We could but it is also being used in the `sellAsset` function so this way we reuse the code

Reply:
After double-checking that `transfer(0)` doesn't raise an exception I decided to use `transfer` instead of a separate method with `.call`